### ISSUE TITLE
Stuck at 'verify this login'

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#14075]
```
SetupEncryptionStore.js?8dc1:71 Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at SetupEncryptionStore.fetchKeyInfo (SetupEncryptionStore.js?8dc1:71)
```

was sitting on a spinner

**Comment by t3chguy** [comment]
This codepath no longer exists on develop as of very recently, maybe the issue too is gone :D?

**Comment by t3chguy** [comment]
Ah this was part of the recovery keys-over-passphrases

**Comment by Vanecx** [comment]
Hello,
Two days ago, I used the new Element on the web browser, I had to verify the login using my Android app, no problem with that except the browser version stayed at "Verify this login" (and I think it only got solved when I refreshed the page).

Today, as the new Linux version is available, I installed it. I again went through the verification process, and I'm again stuck at the "Verify this login" screen that keeps loading. Strangely, when I go in the Preferences, I can see I'm logged in as I can edit my personal preferences (my username is clearly visible etc). **Workaround was to quit the software completely and restart it**, so currently it's working but it might be a problem for other users.
